features/core_trad_register_client.feature

```diff
@@ -16,7 +16,7 @@ Feature: Register a client
     And I click on the css "a#clear-ssm"
     And I wait for the data update
     Then I should see a "System Status" text
-    And I should see a "Software Updates Available" text
+    And I should see a "Software Updates Available" or "System is up to date" text
```

Comment:
Do we have another feature or test where we check if `Software Updates Available` label is shown when updates are available?

If this is the only place where we check it, then I would not change this step and would try wait until the errata information is refreshed or some updates are actually available. 

What do yo think?